android/src/main/kotlin/com.netguru.multiplatformstorage/AndroidStorageProvider.kt

```diff
@@ -25,19 +52,123 @@ internal object AndroidStorageProvider {
      * In other cases, it uses default instance.
      */
     internal fun preferences(name: String? = null): Preferences {
```

Comment:
Set more meaningful name, eg. `generateSecurePreferences`

Reply:
I will keep this name, because of backward compatibility